ui/resetpassword.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE HTML>
```

Comment:
As far as I understand, this page is displayed without any validation, which makes it possible to reach `http://tenant.oae.com/resetpassword/username/` without the secret. Instead, an error should be returned saying "this link has expired" or something equivalent.